cli/assets/templates/assemblyscript/src/wasm4.ts

```diff
@@ -40,29 +40,34 @@ export const BUTTON_DOWN: u8 = 128;
 // └───────────────────────────────────────────────────────────────────────────┘
 
 /** Copies pixels to the framebuffer. */
+// @ts-ignore: decorator
```

Comment:
Is this better than enabling experimentalDecorators in tsconfig.json?

Reply:
decorating in TS not same as in AS, trappings a TS decorators willn't working for declarations.

Normal decorators working for real methods/classes and i not see any errors.

Reply:
> Is this better than enabling experimentalDecorators in tsconfig.json?

TS doesn't support decorators for top-level functions. So this ignores required